src/remoteplayer.h

```diff
@@ -69,6 +69,16 @@ class RemotePlayer : public Player
 		*ratio = m_day_night_ratio;
 	}
 
+	void setTimeOffset(u16 offset)
+	{
+		m_time_offset = offset;
+	}
+
+	void getTimeOffset(u16 *offset)
```

Comment:
is there a reason why this isn't `u16 getTimeOffset() const`?

Reply:
Why not just return the number instead of using a pointer?

Reply:
I used getDayNightRatio as an example because I'm not so good in c++.